### ISSUE TITLE
Supported Bee version is not updated in README

### ISSUE THREAD

**Issue by agazso** [ethersphere/bee-js#677]
The current version still says that is `intended to use with Bee 1.3.0`.

**Comment by AuHau** [comment]
Good catch, thank! I will update the Bee update workflow to automatically update this info as well.

**Comment by bee-worker** [comment]
Release [v5.0.0](https://github.com/ethersphere/bee-js/releases/tag/v5.0.0) addresses this.
